### PR TITLE
Add ARM/Linux rusk wallet build

### PR DESCRIPTION
**Pull request by HDauven**
This PR:
- Fixes an inconsistent behavior in the benchmark CI
- Introduces ARM/Linux Rusk wallet building & `cargo check`

### CODE REVIEW

**Review by Daksh14 — COMMENTED** [review]
The arm build isnt triggered because no changes are made to rusk-wallet. It should be run once before merging into master to avoid broken CI

**Comment by HDauven** [comment]
> The arm build isnt triggered because no changes are made to rusk-wallet. It should be run once before merging into master to avoid broken CI

For the ARM build it doesn't really matter, because it's a manual trigger. I've added ARM/Linux to the build check now. A test run can be found here: https://github.com/dusk-network/rusk/actions/runs/12162377077/job/33919182827

File: .github/workflows/ruskwallet_ci.yml
Comment (Neotamandua) [inline .github/workflows/ruskwallet_ci.yml:1]: nitpick: missing final newline

**Review by Neotamandua — DISMISSED** [review]
LGTM. One really small nitpick

**Pull request merged**